extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/manager/ParticipantManager.java

```diff
@@ -78,8 +78,10 @@ private Boolean processOnboardingInitiated(Participant participant) {
     }
 
     private Boolean processAuthorizing(Participant participant) {
-        var credentialsValid = credentialsVerifier.verifyCredentials();
-        if (credentialsValid) {
+        var credentialsValid = participantVerifier.verifyCredentials(participant.getDid());
+        if (credentialsValid.failed()) {
+            participant.transitionFailed();
+        } else if (credentialsValid.getContent()) {
             participant.transitionAuthorized();
         } else {
             participant.transitionDenied();
```

Comment:
Is the denied case being tested in system tests? Or will it be tested once retrieval of the VCs from the IH is in place?

Reply:
Indeed it is covered in parallel PR